R/opendp/tests/testthat/test-measurements.R

```diff
@@ -67,17 +67,25 @@ test_that("test_gaussian_curve", {
   expect_equal(curve(delta = 1e-3), 0.6880024554878086)
   expect_equal(curve(delta = 1.), 0.)
 
+  expect_equal(curve(epsilon = 0.), 0.1508457845622862)
+  expect_equal(curve(epsilon = 0.6880024554878086), 1e-3)
```

Comment:
Prefer to have expressions that explain how floats like this are derived. Risk of just asserting the current behavior instead of confirming that the calculation is actually correct.

Reply:
I updated to include code here that uses the reverse map. I think it makes more sense to have two implementations and check that they're matching, and point people to the docs of your dependency functions about this. The formula itself is not easy to explain or describe.